### PR TITLE
docs(schedular-service and video-conferencing-service): update README.md

### PR DESCRIPTION
**Pull request by ashutosh-bansal-2136**
## Description

Added dbschema, api details, env and migrations in readme.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)
- [x] Documentation

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [x] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/video-conferencing-service/README.md [hunk 1]
```diff
@@ -1,61 +1,99 @@
...
+  // add VideoConfServiceComponent inside the application class
+  this.component(VideoConfServiceComponent);
   ...
-	```
+  ```
 - Set up a [Loopback4 Datasource](https://loopback.io/doc/en/lb4/DataSource.html) with `dataSourceName` property set to `VideoConfDatasource`. You can see an example datasource [here](#setting-up-a-datasource).
 - Start the application
   `npm start`
 
+### Environment Variables
```
Comment (samarpan-b) [inline services/video-conferencing-service/README.md:60]: Add flow diagram and database schema

File: services/video-conferencing-service/README.md [hunk 2]
```diff
@@ -89,60 +129,78 @@ export class VideoDbDataSource extends juggler.DataSource
...
-## DB migrations
+## Database Schema
+![canva-photo-editor](https://user-images.githubusercontent.com/82804130/126635878-4e39ce2c-e48d-4a67-9c81-6a5f8ee9b70e.png)
+
+## Migrations
 
 The migrations required for this service are processed during the installation automatically if you set the `VIDEOCONF_MIGRATION` or `SOURCELOOP_MIGRATION` env variable. The migrations use [`db-migrate`](https://www.npmjs.com/package/db-migrate) with [`db-migrate-pg`](https://www.npmjs.com/package/db-migrate-pg) driver for migrations, so you will have to install these packages to use auto-migration. Please note that if you are using some pre-existing migrations or database, they may be effected. In such scenario, it is advised that you copy the migration files in your project root, using the `VIDEOCONF_MIGRATION_COPY` or `SOURCELOOP_MIGRATION_COPY` env variables. You can customize or cherry-pick the migrations in the copied files according to your specific requirements and then apply them to the DB.
 
-## APIs available (Currently Vonage is Supported) 
+## APIs available (Currently Vonage is Supported)
```
Comment (samarpan-b) [inline services/video-conferencing-service/README.md:139]: move this to a separate docs.md

File: services/video-conferencing-service/README.md [hunk 3]
```diff
@@ -89,60 +129,78 @@ export class VideoDbDataSource extends juggler.DataSource
...
+This module has the following api endpoints :
+
+`POST /session`
 Used for Creating a session with options such as end to end encryption, archive mode.
-Note: Archving Option cannot be enabled while using end to end encryption, otherwise an Error will be thrown.
+Note: Archiving Option cannot be enabled while using end to end encryption, otherwise an Error will be thrown.
 Successful execution will send a meeting link id which can be used to amend in client url.
 
-### POST /session/{meetingLinkId}/token
+`POST /session/{meetingLinkId}/token`
```
Comment (samarpan-b) [inline services/video-conferencing-service/README.md:152]: move these APIs to openapi.md

**Review by samarpan-b — CHANGES_REQUESTED** [review]

**Review by samarpan-b — APPROVED** [review]

File: services/scheduler-service/README.md [hunk 4]
```diff
@@ -1,69 +1,117 @@
...
 - Events
 - Event Attendee
 - Event Attachment
 
+You can see the database schema [here](#database-schema).
 
-## Install 
+To get started with a basic implementation of this service, see [/sandbox/scheduler-example](https://github.com/sourcefuse/loopback4-microservice-catalog/tree/master/sandbox/scheduler-example).
 
+## Install
```
Comment (akshatdubeysf) [inline services/scheduler-service/README.md:35]: Working/Flow for schedular service readme?

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: services/scheduler-service/README.md (hunk 4 above)
Comment (ashutosh-bansal-2136) [reply services/scheduler-service/README.md:35]: added

**Review by akshatdubeysf — APPROVED** [review]

**Pull request merged**